k8s/namespaces/monitoring/kube-prometheus-stack/patches/dev/cluster-00/mailrelay-alerts-rules.yaml

```diff
@@ -10,27 +10,19 @@ spec:
   "groups":
   - "name": "EximAlerts"
     "rules":
-    - "alert": "eximdown"
-      "annotations":
-        "message": "exim down"
-      "expr": |
-        exim_up == 0
-      "for": "0s"
-      "labels":
-        "severity": "critical"
     - "alert": "EximQueueLength"
       "annotations":
-        "message": "Queue size"
+        "message": "Queue size is above 3"
       "expr": |
         exim_queue > 5
       "for": "5m"
       "labels":
         "severity": "warning"
-    - "alert": "TotalMessageCount"
+    - "alert": "EximQueueLengthNormal"
       "annotations":
```

Comment:
We don't need a warning for queue normal. When alerts are resolved the alertmanager will post a new message anyway.